js/photoFilters.js

```diff
@@ -0,0 +1,120 @@
+const FILTERS = [
+  {
+    name: 'none',
+    style: 'none',
```

Comment:
Возможно нужно вовсе убирать фильтр по ТЗ, но это мелочь, можешь и оставить так
```suggestion
    style: null,
```